### PR TITLE
IRIS-826 add long inventory list

### PR DESCRIPTION
**Pull request by paweltomaszewskisaucelabs**
# One-line summary

> Issue : #1234 (only if appropriate)

## Description
Added a new page with longer inventory (didn't want to break other teams tests)

## Types of Changes
- New feature (non-breaking change which adds functionality)

## Tasks
_List of tasks you will do to complete the PR_
  - [ ] Created Task 1
  - [ ] Created Task 2
  - [ ] To-do Task 3

## Review
_List of tasks the reviewer must do to review the PR_
- [ ] Tests
- [ ] Documentation
- [ ] CHANGELOG

## Deployment Notes
These should highlight any db migrations, feature toggles, etc.

### CODE REVIEW

**Comment by diemol** [comment]
Were you able to build and run the app locally? Last time I tried it didn't work.

**Comment by paweltomaszewskisaucelabs** [comment]
I was able to do that after downgrading node and python version

File: src/pages/Inventory.jsx
Comment (omacranger) [inline src/pages/Inventory.jsx:17]: Perhaps we should have this be `data = InventoryData` with the original import so we default to the current behavior? Would make it require no changes to the existing tests, too.

**Review by omacranger — APPROVED** [review]

File: src/pages/Inventory.jsx
Comment (paweltomaszewskisaucelabs) [reply src/pages/Inventory.jsx:17]: I introduced a second route for an inventory with different set of data. In future will probably add lazy loading feature there and I think it will be easier to test with a prop instead of dealing with routes to define which data should be used. With next iteration I can change prop into `longData: boolean` and deal importing data in only one place.

**Comment by paweltomaszewskisaucelabs** [comment]
@diemol Can you take a look? I need also your approval because you are the code owner.

**Review by diemol — APPROVED** [review]

**Pull request merged**
